static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -381,4 +511,27 @@ const CodeContainer = styled('div')`
   overflow: auto;
 `;
 
+const ValueObjectKey = styled('span')`
+  color: var(--prism-keyword);
+`;
+
+const ValueNull = styled('span')`
+  font-weight: ${p => p.theme.fontWeightBold};
+  color: var(--prism-property);
+`;
+
+const SelectorButton = styled(Button)`
+  background: none;
+  border: none;
+  padding: 0 2px;
+  border-radius: 2px;
+  font-weight: ${p => p.theme.fontWeightNormal};
+  box-shadow: none;
+  font-size: ${p => p.theme.fontSizeSmall};
+  color: ${p => p.theme.subText};
+  margin: 0 ${space(0.5)};
+  height: auto;
+  min-height: auto;
+`;
```

Comment:
might be able to get away with something like `<Button size="zero" borderless>...</Button>` instead of having to write so much new css

Reply:
we could do that! would look like this instead: 
<img width="426" alt="image" src="https://github.com/user-attachments/assets/3e355cf3-96da-4dcb-b5fb-1f95356fa4ea">

Reply:
either way i don't mind!